### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by Ultie0** [pmt-mcpe-fun/web.app#20]
Either I'm not allowed to use it, or you made an oversight.

**Comment by Ad5001** [comment]
You aren't allowed to use the src code exept if we make special contracts.

**Comment by jasonwynn10** [comment]
@Ad5001 you should get PocketMineTools copyrighted

**Comment by NFGamerMC** [comment]
Yes. @jasonwynn10. Although it would not be useful just use a DMCA

**Comment by Ultie0** [comment]
@NFGamerMC it's a good idea to add a license because technically you have to prove that person was in the wrong, or they can counter it saying you probably weren't bothered to add it.

**Comment by Ad5001** [comment]
I'll write a license when I'll have the time to.

**Comment by nathfreder** [comment]
@Ad5001 How about MPL-2.0?

**Comment by Ad5001** [comment]
MPL is too permissive and isn't adapted for a website. We'd need a license to allow `private` instances of the website and allowing contributions while retaining the rights to prevent "fork-and-take"

**Comment by nathfreder** [comment]
@Ad5001 How about dual-licensing? This is a new trend in the open source community where you have a *what you were talking about* licence then a paid one that gives you more permissions.
